### DISCUSSION TITLE
How can I contribute?

### DISCUSSION THREAD

**Discussion by Elektrobomb** [OpenScanEu/OpenScan-PCB#11] [Q&A]
Hi everyone (especially @OpenScanEu),
I found this project and I love the concept.  I feel like some of the electronic design could do with some extra love and I would really like to be able to contribute.  I'm a professional EE so it should be within my wheelhouse to help out :)

How can I get started?

**Comment by probably-Erwins-Cat** [comment]
The simplest way would be contacting me here and joining the discord of the project were stuff of the updated PCBs are discussed: https://discord.gg/TDdyVTa4
